docs/grammar/Solidity.g4

```diff
@@ -1,7 +1,7 @@
 /**
  * Solidity is a statically typed, contract-oriented, high-level language for implementing smart contracts on the Ethereum platform.
  */
-grammar Solidity;
+parser grammar SolidityParser;
```

Comment:
I was not able to debug rules without explicitly saying that this is a parser grammar. However, doing this here has also implications on how `test_antlr_grammar.sh` is executed. Somehow the `SolidityParser.class` is not found anymore, we just name that grammar `Solidity`, so I needed to change that to `SolidityParser`. For `scripts/doc.sh` needs to have `docs/grammar/Solidity.g4` present. To be able to review that easier, I only changed `Solidity.g4` and created a symbolic link `SolidityParser.g4`.